contracts/src/Teleporter/tests/ReceiveCrossChainMessageTests.t.sol

```diff
@@ -238,4 +240,13 @@ contract ReceiveCrossChainMessagedTest is TeleporterMessengerTest {
     function testMessageSentToEOADoesNotExecute() public {
         _receiveMessageSentToEOA();
     }
+
+    function testEmptyReceiptQueue() public {
+        ReceiptQueue.TeleporterMessageReceiptQueue storage receiptQueue = outstandingReceipts[
+            DEFAULT_ORIGIN_CHAIN_ID
+        ];
+
+        assertEq(receiptQueue.first, 0);
+        assertEq(receiptQueue.last, 0);
+    }
```

Comment:
Is this the unit test you mentioned? @minghinmatthewlam

Reply:
can we move this to `ReceiptQueueTests` and add a check that `size()` returns 0, and that `getReceiptAtIndex(0)` reverts for an empty queue.

Can we also add separate unit tests for the new external functions in `ITeleporterMessenger.getReceiptQueueSize()` and `ITeleporterMessenger.getReceiptatIndex()`? These just cover cases like:
- If the chain never received a message yet
- Chain receives a receipt, queue size should increase and the get receipt at index works
- When a message gets sent back to origin chain, queue size decreases and the same index can't be queried for `getReceiptAtIndex` since index represents position relative to`start`